examples/howto/marginalizing-models.ipynb

```diff
@@ -0,0 +1,1527 @@
+{
```

Comment:
<p>Mention that we show this (<code>lp_idx</code>)  in the next example?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/628/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>